src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -0,0 +1,116 @@
+import environment from 'platform/utilities/environment';
+
+// const TOGGLE_VALUES_PATH = '/toggle.json';
+const TOGGLE_VALUES_PATH =
+  '/v0/feature_toggles?features=facilityLocatorShowCommunityCares';
+const TOGGLE_POLLING_INTERVAL = 5000;
+
+let flipperClientInstance;
+
+function FlipperClient({
+  host = environment.API_URL,
+  // host = 'http://localhost:3001',
```

Comment:
Could this comment be removed? I don't see this being changed back from `environment.API_URL`.